scm-ui/ui-components/src/SyntaxHighlighter.tsx

```diff
@@ -56,6 +69,11 @@ const SyntaxHighlighter: FC<Props> = ({ language = defaultLanguage, showLineNumb
     value
   );
 
+  const copy = () => {
+    navigator.clipboard.writeText(value);
+    setCopied(true);
```

Comment:
```suggestion
copyToClipboard(value).then(() => setCopied(true));
```

Why not like this? We have an exported copyToClipBoard function. Just for consistence

Reply:
Didn't know about this one, thx!